E/eccodes/build_tarballs.jl

```diff
@@ -20,17 +20,18 @@ if [[ ${target} = *-mingw* ]] ; then
 else
     atomic_patch -p1 /workspace/srcdir/patches/unix.patch
 fi
+atomic_patch -p1 /workspace/srcdir/patches/kinds.patch
 mkdir build
 cd build
 export CFLAGS="-I${includedir}"
 cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
     -DCMAKE_BUILD_TYPE=Release \
-    -DENABLE_NETCDF=OFF \
+    -DENABLE_NETCDF=ON \
```

Comment:
We have netcdf on a limited number of platforms, because of HDF5, and NetCDF is also plagued by plenty of problems